streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -546,8 +546,9 @@ void removeAllSensors() {
             removeSensor(punctuateTimeSensor);
             removeSensor(taskCreatedSensor);
             removeSensor(tasksClosedSensor);
-            removeSensor(skippedRecordsSensor);
-
+            if (skippedRecordsSensor != null) {
```

Comment:
`skippedRecordsSensor` should not be null, right?

Reply:
it was when I made it lazy. I've fixed it.